JuliaGPU/session2/Videh Raj Nema/a2q4.jl

```diff
@@ -0,0 +1,76 @@
+A=[[2,1];[4,3];[6,5]]
+#=6-element Array{Int64,1}:
+ 2
+ 1
+ 4
+ 3
+ 6
+ 5
+=#
+
+B=[[2,1],[4,3],[6,5]]
+#=3-element Array{Array{Int64,1},1}:
+ [2, 1]
+ [4, 3]
+ [6, 5]
+=#
+sqrt.(A)
+#=6-element Array{Float64,1}:
+ 1.4142135623730951
+ 1.0               
+ 2.0               
+ 1.7320508075688772
+ 2.449489742783178 
+ 2.23606797749979  
+=#
+sort.(A)
+#=ERROR: MethodError: no method matching sort(::Int64)
+Closest candidates are:
+  sort(::AbstractUnitRange) at range.jl:936
+  sort(::AbstractRange) at range.jl:939
+  sort(::SparseArrays.SparseVector{Tv,Ti}; kws...) where {Tv, Ti} at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/SparseArrays/src/sparsevector.jl:1859
+  ...
+Stacktrace:
+ [1] _broadcast_getindex_evalf at ./broadcast.jl:582 [inlined]
+ [2] _broadcast_getindex at ./broadcast.jl:555 [inlined]
+ [3] getindex at ./broadcast.jl:515 [inlined]
+ [4] copy at ./broadcast.jl:790 [inlined]
+ [5] materialize(::Base.Broadcast.Broadcasted{Base.Broadcast.DefaultArrayStyle{1},Nothing,typeof(sort),Tuple{Array{Int64,1}}}) at ./broadcast.jl:756
+ [6] top-level scope at none:0
+=#
+sqrt.(B)
+#=ERROR: MethodError: no method matching sqrt(::Array{Int64,1})
+Closest candidates are:
+  sqrt(::Float16) at math.jl:1018
+  sqrt(::Complex{Float16}) at math.jl:1019
+  sqrt(::Missing) at math.jl:1070
+  ...
+Stacktrace:
+ [1] _broadcast_getindex_evalf at ./broadcast.jl:582 [inlined]
+ [2] _broadcast_getindex at ./broadcast.jl:555 [inlined]
+ [3] getindex at ./broadcast.jl:515 [inlined]
+ [4] copy at ./broadcast.jl:790 [inlined]
+ [5] materialize(::Base.Broadcast.Broadcasted{Base.Broadcast.DefaultArrayStyle{1},Nothing,typeof(sqrt),Tuple{Array{Array{Int64,1},1}}}) at ./broadcast.jl:756
+ [6] top-level scope at none:0
+=#
+sort.(B)
+#=3-element Array{Array{Int64,1},1}:
+ [1, 2]
+ [3, 4]
+ [5, 6]
+=#
```

Comment:
You don't need to put the output in the answer. The question asks you to just explain the output.